include/particle.tcc

```diff
@@ -392,6 +392,65 @@ template <unsigned Tdim, unsigned Tnphases>
 void mpm::Particle<Tdim, Tnphases>::compute_strain(unsigned phase, double dt) {
```

Comment:
OK sorry to go back on this, there is quite a lot of repetition of code here, so let's pass `bbar` as an argument and switch between the `strain_rate`

Reply:
I agree!